lib/pscom4ivshmem/direct_mode/psivshmem_malloc.h

```diff
@@ -0,0 +1,72 @@
+/*
+ * ParaStation
+ *
+ * Copyright (C) 2013 ParTec Cluster Competence Center GmbH, Munich
+ *
+ * All rights reserved.
+ *
+ * Author:	Jens Hauke <hauke@par-tec.com>
```

Comment:
see above. However, you should add an acknowledgment to the original author/file in both cases.